assets/js/modules/search-console/components/dashboard/SearchFunnelWidget/Overview.js

```diff
@@ -180,13 +181,23 @@ const Overview = ( {
 		} )
 	);
 
+	const topAlignedDataBlockClass = {
+		'googlesitekit-data-block--is-top-aligned':
+			! analyticsModuleConnected ||
+			! analyticsModuleActive ||
+			showGoalsCTA,
```

Comment:
Nice catch, and thanks for the comment 👍🏻

Reply:
Let's move this down to the actual `classnames()` call though, instead of creating the variable here.

We tend to place the classnames inline in the component; it's a bit easier to reason about all of the class names at once that way and places them closer to where they're used.

EDIT: Oh, I see that this is used in multiple places, hence the variable. I think a good compromise would be creating this variable as the boolean conditional to show the class, eg:

```js
const alignDataBlockOnTop = ! analyticsModuleConnected ||
	! analyticsModuleActive ||
	showGoalsCTA;
```

then using that conditional inline in each `className` prop, eg:

```js
className={ classnames(
	'googlesitekit-data-block--impressions googlesitekit-data-block--button-1',
	{ 'googlesitekit-data-block--is-top-aligned': alignDataBlockOnTop }
) }
```

It's less DRY, but it's much more explicit, and the conditions, which is the most important thing here to DRY, is still defined in only one place.

Reply:
Thank you for the kind feedback, @tofumatt! Addressed the changes.